course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt

```diff
@@ -1157,6 +1158,59 @@ fun CourseDatesBannerTablet(
     }
 }
 
+@Composable
+fun DatesShiftedSnackBar(
+    showAction: Boolean = false,
+    onViewDates: () -> Unit? = {},
+    onClose: () -> Unit? = {},
+) {
+    Snackbar(
+        modifier = Modifier.padding(16.dp),
+        backgroundColor = MaterialTheme.appColors.background
+    ) {
+        Column(modifier = Modifier.padding(4.dp)) {
+            Box {
+                Text(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .align(Alignment.CenterStart),
+                    text = stringResource(id = org.openedx.core.R.string.core_dates_shift_dates_successfully_title),
```

Comment:
please move org.openedx.core.R to import `import org.openedx.core.R as CoreR`

Reply:
moved.